src/librescore-link.ts

```diff
@@ -6,7 +6,7 @@ const _getLink = (scorepack: string) => {
   return `https://librescore.org/score/${scorepack}`
 }
 
-export const getLibreScoreLink = async (scoreinfo: ScoreInfo, _fetch = getFetch()): Promise<string> => {
+export const getLibreScoreLink = async (scoreinfo: ScoreInfo, isMsdl: Boolean, _fetch = getFetch()): Promise<string> => {
```

Comment:
```suggestion
export const getLibreScoreLink = async (scoreinfo: ScoreInfo, isMsdl: boolean, _fetch = getFetch()): Promise<string> => {
```

[The TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object) suggests using lowercase `boolean` for primitive type.